catalog-management/src/main/java/com/c12e/cortex/examples/catalog/RailedCommand.java

```diff
@@ -19,22 +19,22 @@
 import com.c12e.cortex.phoenix.ProfileSchema;
 import com.c12e.cortex.phoenix.spec.*;
 import com.c12e.cortex.profiles.CortexSession;
-import com.fasterxml.jackson.core.JsonParser;
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.DeserializationContext;
-import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
-import com.fasterxml.jackson.databind.json.JsonMapper;
-import com.fasterxml.jackson.databind.module.SimpleModule;
-import com.fasterxml.jackson.module.kotlin.KotlinModule;
-import com.jayway.jsonpath.DocumentContext;
-import com.jayway.jsonpath.JsonPath;
-import com.jayway.jsonpath.Option;
-import com.jayway.jsonpath.TypeRef;
-import com.jayway.jsonpath.spi.json.JacksonJsonProvider;
-import com.jayway.jsonpath.spi.json.JsonProvider;
-import com.jayway.jsonpath.spi.mapper.JacksonMappingProvider;
-import com.jayway.jsonpath.spi.mapper.MappingProvider;
+import com.c12e.shadow.com.fasterxml.jackson.core.JsonParser;
+import com.c12e.shadow.com.fasterxml.jackson.core.JsonProcessingException;
+import com.c12e.shadow.com.fasterxml.jackson.databind.DeserializationContext;
+import com.c12e.shadow.com.fasterxml.jackson.databind.JsonNode;
+import com.c12e.shadow.com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.c12e.shadow.com.fasterxml.jackson.databind.json.JsonMapper;
+import com.c12e.shadow.com.fasterxml.jackson.databind.module.SimpleModule;
+import com.c12e.shadow.com.fasterxml.jackson.module.kotlin.KotlinModule;
+import com.c12e.shadow.com.jayway.jsonpath.DocumentContext;
+import com.c12e.shadow.com.jayway.jsonpath.JsonPath;
+import com.c12e.shadow.com.jayway.jsonpath.Option;
+import com.c12e.shadow.com.jayway.jsonpath.TypeRef;
+import com.c12e.shadow.com.jayway.jsonpath.spi.json.JacksonJsonProvider;
+import com.c12e.shadow.com.jayway.jsonpath.spi.json.JsonProvider;
+import com.c12e.shadow.com.jayway.jsonpath.spi.mapper.JacksonMappingProvider;
+import com.c12e.shadow.com.jayway.jsonpath.spi.mapper.MappingProvider;
```

Comment:
This seems fine, but should we be recommending that Users of the SDK import shaded modules? Wouldn't the user declare this Jackson stuff in their Project dependencies, and there be no version conflicts?

Reply:
Yes, but I also don't want to get into maintaining versions in the examples repo. So I figured I'd take them from the profiles-sdk JAR. If a user was using this and wanted their own versions, they could.